common/src/main/java/com/microsoft/identity/common/internal/cache/BrokerOAuth2TokenCache.java

```diff
@@ -551,7 +557,8 @@ public ICacheRecord load(@NonNull final String clientId,
     @Override
     public List<ICacheRecord> loadWithAggregatedAccountData(@NonNull final String clientId,
                                                             @Nullable final String target,
-                                                            @NonNull final AccountRecord account) {
+                                                            @NonNull final AccountRecord account,
+                                                            @NonNull final AbstractAuthenticationScheme authScheme) { // TODO implement!
```

Comment:
do we need this todo?

Reply:
Also done, will remove - thanks

Reply:
Resolved in 4ea69de4f7fd98b6aa4cbff086bf76e341791d9d